_docs/00-getting-started.md

```diff
@@ -5,23 +5,29 @@ layout: docs
 permalink: /docs/getting-started.html
 ---
 
-## Get Infer
+## How to Download and Install Infer
 
 You can use Homebrew (Mac only), our binary releases, build infer from
-source, or use our Docker image.
+source, or use our Docker image. As of April 2019 (version 0.16.0),
+**it will require about 1.2 GB of disk space.**
```

Comment:
Which of these 3 installations methods does 1.2 GB correspond to? I think that number should be moved to the relevant section of this document. Installing from source requires a lot more than 1.2 GB to compile clang for instance.

Reply:
I can clarify this.